README.md

```diff
@@ -50,8 +50,10 @@ pip install -e ".[dev]"
 To run all tests, use the following command:
 
 ```sh
-python setup.py test
+make test
 ```
+  
+ Some of the tests require Docker running on your machine before you launch the tests.
```

Comment:
```suggestion
 Some of the tests require Docker running on your machine before running them.
```

I put it as suggestion but know that suggestion are not signed of, so you might need to do the modification on your own of squash the suggested commits 😇